src/main/java/com/dicemc/corruptedlands/CorruptedLandMod.java

```diff
@@ -121,6 +126,10 @@ else if (player.getHealth() > Config.CORRUPTION_EFFECT_POWER.get()){
 					if (bs.getBlock() instanceof ICorrupted) {
 						event.getEntityLiving().heal(Config.CORRUPTION_EFFECT_POWER.get());
 					}
+					if(calyxCap != null && event.getEntityLiving() instanceof InfectedEntity && bs.getBlock() instanceof ICorrupted){
```

Comment:
Have you tested CL with the changes and IC not installed?  This line looks safe, but I want to be sure.

Reply:
I have tested it with immortuos both removed in dev environment and out of dev environment without fail.

Reply:
Awesome!  i'll merge this then.